assets/blocks/reader-registration/edit.js

```diff
@@ -137,54 +137,62 @@ export default function ReaderRegistrationEdit( {
 								{ inFlight && <Spinner /> }
 								{ ! inFlight && ! Object.keys( listConfig ).length && (
 									<div style={ { marginBottom: '1.5rem' } }>
-										<Notice isDismissible={ false } status="error">
-											{ __( 'You must enable lists for subscription.', 'newspack-newsletters' ) }
-										</Notice>
-									</div>
-								) }
-								<ToggleControl
-									label={ __( 'Display list description', 'newspack-newsletters' ) }
-									checked={ displayListDescription }
-									disabled={ inFlight }
-									onChange={ () =>
-										setAttributes( { displayListDescription: ! displayListDescription } )
-									}
-								/>
-								<ToggleControl
-									label={ __( 'Hide newsletter selection and always subscribe', 'newspack' ) }
-									checked={ hideSubscriptionInput }
-									disabled={ inFlight || lists.length !== 1 }
-									onChange={ () =>
-										setAttributes( { hideSubscriptionInput: ! hideSubscriptionInput } )
-									}
-								/>
-								{ lists.length < 1 && (
-									<div style={ { marginBottom: '1.5rem' } }>
-										<Notice isDismissible={ false } status="error">
-											{ __( 'You must select at least one list.', 'newspack-newsletters' ) }
-										</Notice>
+										{ __(
+											'To enable newsletter subscription, you must enable subscription lists on Newspack Newsletters.',
```

Comment:
minor: maybe "you must enable at least one Subscription List..."?

Reply:
I decided to change a bit more of the phrasing there. WDYT?

<img width="274" alt="image" src="https://user-images.githubusercontent.com/820752/208470879-e2eeb47e-7eeb-4bd1-b39e-775374feb2b2.png">

The use of "configure" in the paragraph should lead to the link.

Reply:
Looks good